package/upgrade/lib.sh

```diff
@@ -210,3 +210,54 @@ detect_upgrade()
 
   UPGRADE_PREVIOUS_VERSION=$(echo "$upgrade_obj" | yq e .status.previousVersion -)
 }
+
+upgrade_addon()
+{
+  local name=$1
+  local namespace=$2
+  local version=$3
+
+  version=$(cat ./addons/${name}.yaml | yq e .spec.version)
```

Comment:
Need to use the directory in `/usr/local/share`

Reply:
fixed.